Builder/SolariumSelectQueryBuilder.php

```diff
@@ -71,33 +64,35 @@ public function buildSolariumQueryFromGeneric(GenericSelectQuery $query)
         }
 
         //determine whether we are using the facet values for an underlying "base" (recorded) query
-        $shouldUseFacetValuesForRecordedQuery = (bool) $this->hasSearchContext()
-            && $query instanceof RecordableSelectQueryInterface
+        $shouldUseFacetValuesForRecordedQuery = (bool) $query instanceof RecordableSelectQueryInterface
```

Comment:
Yeah - maybe I should add the whole check to the 'ResolvedSelectQueryInterface'? I still can't see why the context is relevant though...